### ISSUE TITLE
User request: hide/show elements on the home page

### ISSUE THREAD

**Issue by Siderite** [Siderite/lichessTools#491]
Elements of the home page:
  - side (streams, spotlights, timeline)
  - TV
  - table (start,counters)
  - app (whole thing, just certain tabs)
  - puzzle
  - blog
  - leaderboard
  - winners
  - tournaments
  - feed
  - support (maybe this should not be an option to remove, but just minify)
  - about (also this should always be visible)

**Comment by Siderite** [comment]
The arrangement of the elements is done as a grid:
```
.lobby {
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: repeat(2, fit-content(0)) auto repeat(6, fit-content(0));
  grid-template-areas:  "side   app     app     table"
                                     "tv     blog    blog    puzzle"
                                     "tv     support support puzzle"
                                     "feed   feed    tours   tours"
                                     "feed   feed    leader  winner"
                                     "about  about   leader  winner";
}
```
And this is just for a specific screen width configuration. (BTW, the last repeat should be repeat(3,... )
This does not lend itself easily to hiding individual elements. Instead, three solutions:
- create some algorithm that would create the perfect arrangement of displayed elements depending on screen width
- change the layout completely to support dynamic element display
- create a few separate layouts that would cover most preferences

Obviously the first two require a lot of research and work and the design is brittle. Any change by Lichess would either not be expressed or it would break something.

This only leaves the third, which can be implemented as themes based on individual requests.

**Comment by Dboingue** [comment]
knap sack problem the first one.  (or Tetris?)

I think lichess got wise in partitioning things that should flock together. The X-mas tree decorations for the not-the-rest-of-us, and the wanna-bees  (hah! get that! spell checker!), for example.

There could be the player only theme.  The forum only theme (wait what?)
The tournament is the only chess there is, theme (offer and glorious feedback thingies).
The study theme (also wait, what?).

The TV theme. The puzzle theme. ok I am not helping....

**Comment by Siderite** [comment]
Implemented in v2.3.41
